src/Utils/Rating/index.js

```diff
@@ -0,0 +1,8 @@
+import { Rate } from 'antd'
+const Rating = (rating) => {
+  return (
+    <div><Rate disabled allowHalf defaultValue={0} value={rating}></Rate></div>
```

Comment:
Please, use Rate component below div

Reply:
thankyou for the fix 2313f342525894ae95ccbcbbe4d0766e8bf1e34a